magicctapipe/scripts/lst1_magic/semi_automatic_scripts/dl1_production.py

```diff
@@ -76,7 +76,7 @@ def config_file_gen(target_dir, noise_value, NSB_match, source_name, config_gen)
         LST_config["increase_nsb"]["extra_bias_in_dim_pixels"] = noise_value[2]
         LST_config["increase_nsb"]["extra_noise_in_bright_pixels"] = noise_value[1]
     conf = {
```

Comment:
similar like for the coincidence, we could save directly the config read from config_file.
for MCs you would modify it in the meantime in the lines 74-77, but those lines will be either way gone with MC part removed

Reply:
I removed the change in the noise values. Then #243 should completely remove the MC

Reply:
ok, but if you removed adding noise already in this PR, NSB_match does not make sense, and the whole MC part is missing will be missing one element after another. 
Actually I was not suggesting to remove the added noise already in this PR, sorry if it was not clear from my previous comment. Maybe best that you add this back (sorry again) and then we merge it.

Reply:
No worries, my fault. I will add it again and check if the 'no MC' PR needs a similar fix in the config